remote-config.jsonc

```
@@ -123,6 +123,15 @@
                 },
                 {
                     "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
+                },
+				{
+                    "message": "Has one of your containers suddenly crashed? Is `ddev describe` returning a black file? Do not worry, type `ddev logs --follow --time` for insights into such issue"
+                },
+				{
+                    "message": "DDEV.com is open source and the perfect place to share a case study, innovative uses of DDev, or your custom-made guides and recipes."
```

I’d use lowercase for a domain name, and always refer to “DDEV” the product name.

```suggestion
                    "message": "ddev.com is open source and the perfect place to share a case study, innovative uses of DDEV, or your custom-made guides and recipes."
```